packages/patternfly-3/package.json

```diff
@@ -71,7 +71,23 @@
       "grunt-jekyll",
       "grunt-sync",
       "font-awesome",
-      "patternfly"
+      "patternfly",
+      "angular-patternfly",
+      "animate.css",
+      "bootstrap",
+      "bootstrap-combobox",
+      "bootstrap-datepicker",
+      "bootstrap-select",
+      "clipboard",
+      "datatables",
+      "font-awesome",
+      "google-code-prettify",
+      "jquery-match-height",
+      "jquery.scrollto",
+      "patternfly",
```

Comment:
`patternfly` and `fontawesome` are here twice, but that's ok.

Reply:
oh thanks! i copied the lines from dependencies and overlooked that some were already in here. Maybe the next PR can address this?

Reply:
Yep!